modules/imgproc/src/undistort.cpp

```diff
@@ -120,12 +135,129 @@ class initUndistortRectifyMapComputer : public cv::ParallelLoopBody
             else
                 CV_Assert(m1 != NULL);
 
-    #if CV_TRY_AVX2
+#if CV_TRY_AVX2
             if( useAVX2 )
                 j = cv::initUndistortRectifyMapLine_AVX(m1f, m2f, m1, m2,
```

Comment:
What is about performance difference between w.u.i below and this "AVX" code branch?

Reply:
I suppose AVX2 branch should be executed for AVX2 baseline. Also I've tested performance against CPU_BASELINE=AVX2 and CPU_DISPATCH=AVX2 and got 15% performance improvement. So I think average performance gain is about 10% due to manual cycle unrolling and a few reused constant values.
I think it is reasonable to replace AVX2 branch with dynamic dispatching of w.u.i code. Does it make sense to make this change a part of the PR?

Reply:
It make sense to avoid code duplication if we archiving similar performance.